common/src/main/java/com/microsoft/identity/common/internal/controllers/LoadAccountCommand.java

```diff
@@ -0,0 +1,107 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.internal.controllers;
+
+import android.accounts.AuthenticatorException;
+import android.accounts.OperationCanceledException;
+import android.os.RemoteException;
+import android.support.annotation.NonNull;
+
+import com.microsoft.identity.common.exception.ClientException;
+import com.microsoft.identity.common.internal.dto.AccountRecord;
+import com.microsoft.identity.common.internal.request.OperationParameters;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+public class LoadAccountCommand {
```

Comment:
I would expect multiple classes called `Command` to share a superclass other than `java.lang.Object`.

These classes should have an `abstract` superclass defining their shared methods and constructors

Reply:
Preferably... there should be an `interface` that's implemented by the `abstract` class, then subclasses `extend` the superclass

Reply:
Then you can write the shared javadoc once ;)

Reply:
Consider adding `@Nullable` & `@NonNull` decorations to these methods too. [So long as the subclasses adhere to this behavior](https://en.wikipedia.org/wiki/Liskov_substitution_principle)